source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -1,871 +1,914 @@
+The following examples are designed to work together and in order. For
+simplicity, we use a setup that can be easily reproduced using a stock |CL|
+install. We'll use:
 
-.. code-block:: bash
+* A stock installation of |CL| with all `Prerequisites`_.
+* A web server that comes with |CL| to host the content updates.
+  (If you haven't already, `Set up a nginx web server for mixer`_.)
+* A simple VM that will update against the locally produced content created in
+  Example 2.
 
-   mixer init
+Example 1: First time set up
+----------------------------
 
-This command initializes your workspace so you can make a mix at version 10
-based on the latest released upstream |CL| version. If a :file:`builder.conf`
-file is not already present in your workspace, mixer creates a default
-configuration file. Additionally, mixer creates several version and tracking
-files, and two bundle directories: :file:`local-bundles` and
-:file:`upstream-bundles`.
+This example shows the basic steps for first time setup of mixer for a new mix.
 
-If you wish to start with a different version of upstream |CL| or a
-different initial mix version, you can specify these options as flags.
-For example:
+#. Create an empty directory to use as a workspace for mixer:
 
-.. code-block:: bash
+   .. code-block:: bash
 
-   mixer init --upstream-version 21060 --mix-version 10
+      mkdir ~/mixer
 
+#. In your mixer workspace, generate an initial mix based on the latest upstream
+   |CL| version, with minimum bundles:
 
-Additionally, to build a mix with your own custom RPMs, use the optional
-*--local-rpms* flag, for example:
+   .. code-block:: bash
 
-.. code-block:: bash
+      cd ~/mixer
+      mixer init
 
-   mixer init --local-rpms
+   Note in the initialization output, that your initial mix version is set to
+   10 and that the minimum bundles have been added.
 
-This command creates the :file:`local-yum` and :file:`local-rpms`
-directories in your mix workspace and adds their paths to the generated
-:file:`builder.conf`. If the configuration file already exists, you must add
-the paths manually. For more information on using these directories or
-setting them up manually, see `Create or locate RPMs for the mix`_.
+#. Edit :file:`builder.conf` to set the value of CONTENTURL and VERSIONURL to
+   the IP address of your content server:
 
-If all upstream |CL| bundles will be part of the mix, you can easily add 
-them all during initialization with the optional *--all-upstream* flag. For example:
+   .. code-block:: bash
 
-.. code-block:: bash
+      nano builder.conf
 
-   mixer init --all-upstream
+   For example:
 
-Finally, you may want to track the contents of your mixer workspace with a
-git repository. This is a great way to track changes to your mix's content
-or to revert to earlier versions if something goes wrong. Mixer can set this
-up automatically with the optional *--git* flag, for example:
+   .. code-block:: console
+
+      CONTENTURL=http://192.168.25.52
```

Comment:
Lines 244 and 245: When I did this, builder.conf required that the format be like this:

CONTENTURL = "http://192.168.25.52"